client/packages/common/src/utils/formatters/formatters.ts

```diff
@@ -13,7 +13,8 @@ export const Formatter = {
     else return null;
   },
   naiveDateTime: (date?: Date | null): string | null => {
-    if (date && isValid(date)) return format(date, "yyyy-MM-dd'T'HH:mm:ss");
+    if (date && isValid(date))
+      return format(date, "yyyy-MM-dd'T'HH:mm:ss+mm:mm");
```

Comment:
I wouldn't say this is a naive date. We needed an `Date.toISOString()` for the api call so I would keep the js name and also use the build in `toISOString()` method.